.changeset/wise-walls-crash.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": minor
```

Comment:
question: Did you read what I posted on QA channel about mark changset as patch when there is no blocker to backport them to current version?